### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by lukemciKainos**

### CODE REVIEW

File: Doras/src/app/app.module.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { Table1Component } from './table1/table1.component';
+import { PopupsComponent } from './popups/popups.component';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    CarouselTabsComponent,
+    CarouselComponent,
+    RolesComponent,
+    Table1Component,
```
Comment (micalexis) [inline Doras/src/app/app.module.ts]: Table1Component? please come up with a better name than that

File: Doras/src/app/carousel-tabs/carousel-tabs.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@Component({
+  selector: 'doras-carousel-tabs',
+  templateUrl: './carousel-tabs.component.html',
+  styleUrls: ['./carousel-tabs.component.css'],
+})
+export class CarouselTabsComponent implements OnInit {
+
+  constructor() { }
+
+  tabs : string[] = [
```
Comment (micalexis) [inline Doras/src/app/carousel-tabs/carousel-tabs.component.ts:13]: why is this not stored in the DB? tabs is also not a good name! what does the tab represent? capability? if so, call it that!

File: Doras/src/app/carousel/carousel.component.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<div class='container-fluid' data-interval="false">
+    <div class='col-12'>
+        <ngb-carousel data-interval="false">
+            <ng-template ngbSlide>
```
Comment (micalexis) [inline Doras/src/app/carousel/carousel.component.html]: this should not be hardcoded, you should use the ngFor directive to loop over the list from your component!

File: Doras/src/app/carousel/carousel.component.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { NgbCarouselConfig } from '@ng-bootstrap/ng-bootstrap';
+@Component({
+  selector: 'app-carousel',
+  templateUrl: './carousel.component.html',
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: ['./carousel.component.css'],
+  styles: [`
```
Comment (micalexis) [inline Doras/src/app/carousel/carousel.component.ts:10]: why have not put this style in the `carousel.component.css` file??

File: Doras/src/app/carousel/carousel.component.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    .carousel-indicators{
+      color: chartreuse;
+      height: 10px;
+    }
+  `]
+})
+export class CarouselComponent implements OnInit {
+
+  constructor(config: NgbCarouselConfig) {
```
Comment (micalexis) [inline Doras/src/app/carousel/carousel.component.ts]: is this supposed to be injected? where is the access modifier!

File: Doras/src/app/roles/roles.component.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<div class='container-fluid' data-interval="false">
+    <div class='col-12'>
+        <ngb-carousel data-interval="false">
+            <ng-template ngbSlide>
+                <app-table1></app-table1>
```
Comment (micalexis) [inline Doras/src/app/roles/roles.component.html]: come up with a better selector name! `app-table1` is not good naming convention!

File: Doras/src/app/table1/table1.component.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<table class="table1">
```
Comment (micalexis) [inline Doras/src/app/table1/table1.component.html]: change the class name!

File: Doras/src/app/table1/table1.component.html [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    
+        <tr>
+            <th class="th1" scope="col">Apprentice</th>
+            <th align="center" scope="col">Trainee</th>
+            <th align="center" scope="col">Associate</th>
+        </tr>
+    
+    
+        <tr>
```
Comment (micalexis) [inline Doras/src/app/table1/table1.component.html]: what is this supposed to do?

File: ExpressAPI/db.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+})
+
+db.connect(function(err) {
+    if (err) throw err
+    console.log('Connected to MySQL')
+})
+
+exports.getBand = function(callback) {
+    db.query(
+        "SELECT * FROM Band",
```
Comment (micalexis) [inline ExpressAPI/db.js]: remove `*`, explicitly define the column names! its bad practice

File: ExpressAPI/db.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+        function(err, rows) {
+            if (err) throw err
```
Comment (micalexis) [inline ExpressAPI/db.js]: where are the brackets?

**Review by micalexis — CHANGES_REQUESTED** [review]
address the comments

File: Doras/src/app/carousel/carousel.component.ts (hunk 4 above)
Comment (lukemciKainos) [reply Doras/src/app/carousel/carousel.component.ts:10]: The slide component is a separate element and is not build with the rest of the html file. This means the styles must be developed in the component.

File: Doras/src/app/table1/table1.component.html (hunk 8 above)
Comment (lukemciKainos) [reply Doras/src/app/table1/table1.component.html]: Spacing fixed

File: Doras/src/app/carousel-tabs/carousel-tabs.component.ts (hunk 2 above)
Comment (lukemciKainos) [reply Doras/src/app/carousel-tabs/carousel-tabs.component.ts:13]: to be completed at a later stage

Comment (lukemciKainos) [reply Doras/src/app/carousel-tabs/carousel-tabs.component.ts:13]: @joshuahigginson

File: Doras/src/app/carousel/carousel.component.html (hunk 3 above)
Comment (lukemciKainos) [reply Doras/src/app/carousel/carousel.component.html]: @lukemciKainos

File: ExpressAPI/db.js (hunk 9 above)
Comment (lukemciKainos) [reply ExpressAPI/db.js]: @joshuahigginson

File: ExpressAPI/db.js (hunk 10 above)
Comment (lukemciKainos) [reply ExpressAPI/db.js]: @joshuahigginson

**Pull request merged**